app/HutchManager/Middleware/ConfigCookieMiddleware.cs

```diff
@@ -1,27 +1,35 @@
 using System.Text.Json;
+using HutchManager.Constants;
 using HutchManager.Models;
 using HutchManager.Services;
+using Microsoft.FeatureManagement;
 
 namespace HutchManager.Middleware;
 
 public class ConfigCookieMiddleware
 {
 
   public static readonly string ConfigCookieName = ".HutchManager.Config";
-
   private readonly RequestDelegate _next;
+  private readonly IConfiguration _config;
 
   public ConfigCookieMiddleware(
+    IConfiguration config,
     RequestDelegate next
     )
   {
     _next = next;
+    _config = config;
```

Comment:
don't use `IConfiguration` directly, use `IOptions<TOptionsModel>`.

You can look at Email sending for examples of this pattern.

Docs are here, if helpful: https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-6.0